openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -0,0 +1,217 @@
+import os
+import clique
+
+import maya.cmds as cmds
+import mtoa.ui.arnoldmenu
+
+from openpype.settings import get_project_settings
+from openpype.pipeline import (
+    load,
+    get_representation_path
+)
+from openpype.hosts.maya.api.lib import (
+    unique_namespace, get_attribute_input, maintained_selection
+)
+from openpype.hosts.maya.api.pipeline import containerise
+
+
+def is_sequence(files):
+    sequence = False
+    collections, remainder = clique.assemble(files)
+    if collections:
+        sequence = True
+
+    return sequence
+
+
+class ArnoldStandinLoader(load.LoaderPlugin):
+    """Load as Arnold standin"""
+
+    families = ["ass", "animation", "model", "proxyAbc", "pointcache"]
+    representations = ["ass", "abc"]
+
+    label = "Load as Arnold standin"
+    order = -5
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name, namespace, options):
+        version = context['version']
+        version_data = version.get("data", {})
+
+        self.log.info("version_data: {}\n".format(version_data))
+
+        asset = context['asset']['name']
+        namespace = namespace or unique_namespace(
+            asset + "_",
+            prefix="_" if asset[0].isdigit() else "",
+            suffix="_",
+        )
+
+        # Root group
+        label = "{}:{}".format(namespace, name)
+        root = cmds.group(name=label, empty=True)
+
+        # Set color.
+        project_name = context["project"]["name"]
+        settings = get_project_settings(project_name)
+        colors = settings['maya']['load']['colors']
```

Comment:
This could be something like:
```
color = settings['maya']['load']['colors'].get("ass")
```

Or if we know the default colour we could do something like:
```py
r, g, b = settings['maya']['load']['colors'].get(
    "ass",
    (0,0,0) # this should be the default
)
cmds.setAttr(root + ".useOutlinerColor", True)
cmds.setAttr( root + ".outlinerColor", r, g, b)
```